build-linux-wheels.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+# Checking for /.dockerenv is a hacky way to determine whether or not we're
+# already running in a Docker container. Note that this is not guaranteed to
+# exist in all versions and drivers and may need to be changed later.
+if [ ! -e /.dockerenv ]; then
+    docker build -f Dockerfile.wheels -t rasterio-wheelbuilder .
+    exec docker run -v `pwd`:/io rasterio-wheelbuilder "$@"
+fi
```

Comment:
Would this be better off as two scripts?

* one that would build the wheels on the linux container
* one that would trigger the build from the docker host? (maybe even better as a Makefile target?)